docs/service-types/types-static-site.md

```diff
@@ -0,0 +1,34 @@
+---
+id: static-site
+title: Static Site
+---
+
+KintoHub supports hosting of websites built with static site generator frameworks. Currently, only websites built in NodeJS language are supported.
+
+![Static](/img/anatomy/static-service.png)
+
+Very soon KintoHub will provide a feature to allow users to integrate and deploy continuously with CI/CD. If you are keen to see this feature in action, vote for it [here](https://feedback.kintohub.com/feature-requests/p/cicd).
+
+
+![CI](/img/anatomy/integration.png)
+
+## When to use a static site?
+
+Static websites are a good choice if performance and security are critical to you. 
```

Comment:
comment - Change this to "Static website are a good choice when you want to couple them with content delivery networks (CDN) or have a website that does **note** require a port.  If your website requires a port, checkout **Web Apps** <-- give it a link.

Reply:
Done